src/purview.ts

```diff
@@ -1087,6 +1092,22 @@ function unalias(id: string, root: ConnectedRoot): string {
   return id
 }
 
+async function saveRootsState(wsState: WebSocketState): Promise<void> {
```

Comment:
I wanna call this `saveRootsAndCSSState` but I don't like the "And". I also don't want to break this into 2 methods